x-pack/plugins/ml/public/application/services/ml_server_info.test.ts

```diff
@@ -59,4 +60,30 @@ describe('ml_server_info', () => {
       done();
     });
   });
+
+  describe('cloud extract deployment ID', () => {
+    const cloudIdWithDeploymentName =
+      'cloud_message_test:ZXUtd2VzdC0yLmF3cy5jbG91ZC5lcy5pbyQ4NWQ2NjZmMzM1MGM0NjllOGMzMjQyZDc2YTdmNDU5YyQxNmI1ZDM2ZGE1Mzk0YjlkYjIyZWJlNDk1OWY1OGQzMg==';
+
+    const cloudIdWithOutDeploymentName =
+      ':ZXUtd2VzdC0yLmF3cy5jbG91ZC5lcy5pbyQ4NWQ2NjZmMzM1MGM0NjllOGMzMjQyZDc2YTdmNDU5YyQxNmI1ZDM2ZGE1Mzk0YjlkYjIyZWJlNDk1OWY1OGQzMg==';
+
+    const badCloudId = 'cloud_message_test:this_is_not_a_base64_string';
+
+    it('cloud ID with deployment name', () => {
```

Comment:
test suit sentence usually starts with `should`, e.g. `should extract cloud id when deployment name is presented`